woocommerce/woocommerce-gutenberg-products-block
# Mini Cart: Style doesn't apply to quantity input

Fixes #6029

This PR styles the quantity input using the text color set in the Mini Cart Contents block. The style in the editor and frontend are handled differently. I added a comment explaining the reason for the difference in code.





### Screenshots

**Editor:**
<img width="1372" alt="image" src="https://user-images.githubusercontent.com/5423135/159656148-9d011295-84f2-4d9f-bedc-2abe1722bac4.png">

**Frontend:**
<img width="1372" alt="image" src="https://user-images.githubusercontent.com/5423135/159656192-664dee7d-5c72-42f8-8202-a3ab7dc77935.png">




### Testing

#### Automated Tests
### Manual Testing

How to test the changes in this Pull Request:

1. Edit Mini Cart Contents block.
2. Set the text and background color.
3. See the input style updated accordingly.
4. Save changes.
5. See the style applied on the front-end as well.

### User Facing Testing
These are steps for user testing (where "user" is someone interacting with this change that is not editing any code).
